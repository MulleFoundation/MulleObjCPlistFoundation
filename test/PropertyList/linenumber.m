//
//  main.m
//  archiver-test
//
//  Created by Nat! on 19.04.16.
//  Copyright Â© 2016 Mulle kybernetiK. All rights reserved.
//
#import <MulleObjCPlistFoundation/MulleObjCPlistFoundation.h>

// there is an error in line #25
static char  text[] =
#include "Movies.plist.inc"
;


int   main(int argc, const char * argv[])
{
   NSData                 *data;
   id                     decoded;
   NSPropertyListFormat   format;

   format = MullePropertyListLooseFormat;
   @try
   {
      data    = [NSData dataWithBytes:text
                               length:sizeof( text)];
      decoded = [NSPropertyListSerialization propertyListFromData:data
                                                 mutabilityOption:0
                                                           format:&format
                                                 errorDescription:NULL];
      mulle_printf( "%@\n", decoded ? decoded : [NSError mulleExtract]);
   }
   @catch( NSException *exception)
   {
      mulle_fprintf( stderr, "exception: %@\n", [exception reason]);
   }

   return( 0);
}
