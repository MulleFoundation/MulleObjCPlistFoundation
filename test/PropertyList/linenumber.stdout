MulleObjCUTF8StreamReader: -1 { {
    NSLocalizedDescription = "<input>:25:error stray '#' (35) in input (needs to be quoted)";
} }
